frameworks/hdfs/integration/tests/defaults.py

```diff
@@ -0,0 +1,12 @@
+import os
```

Comment:
Maybe just merge the contents of this file into task_utils.py? Doesn't feel like there's a lot of reason to separate them